scripts/vits/inflect/micro_v2/generate_samples.py

```
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# Copyright    2026  Xiaomi Corp.        (authors: Fangjun Kuang)
+"""
+Generate samples for
+https://k2-fsa.github.io/sherpa/onnx/tts/all/
+"""
+
+import os
```

Flake8 (7.3.0)

[error] 'os' imported but unused

(F401)